src/base/memory_monitor_common.h

```diff
@@ -29,19 +29,19 @@
 
 typedef enum mmon_module_id
 {
-  /* TODO: this dummy modules will be changed when heap module is registered */
   MMON_MODULE_ALL = -1,
-  MMON_MODULE_DUMMY,
-  MMON_MODULE_LONG_DUMMY,
+  MMON_MODULE_HEAP,
+  MMON_MODULE_COMMON,
+  MMON_MODULE_OTHERS,
```

Comment:
```suggestion
  MMON_MODULE_COMMON,
  MMON_MODULE_HEAP,
  /* Add new modules before this */
  MMON_MODULE_OTHERS,
```

Maybe same opinion with above (in utils.msg)
And it would be better to leave comments on where to add new modules.

Reply:
Done.